x-pack/plugins/ingest_pipelines/public/application/components/pipeline_processors_editor/components/processor_settings_form/processors/common_fields/processor_type_field.tsx

```diff
@@ -39,27 +47,21 @@ const typeConfig: FieldConfig = {
 
 export const ProcessorTypeField: FunctionComponent<Props> = ({ initialType }) => {
   return (
-    <FormRow
-      title={i18n.translate('xpack.ingestPipelines.pipelineEditor.typeFieldTitle', {
-        defaultMessage: 'Type',
-      })}
-    >
-      <UseField config={typeConfig} path={'type'} defaultValue={initialType}>
-        {typeField => {
-          return (
-            <EuiComboBox
-              onChange={([selected]) => typeField.setValue(selected?.value)}
-              selectedOptions={
-                typeField.value
-                  ? [{ value: typeField.value as string, label: typeField.value as string }]
-                  : []
-              }
-              singleSelection={{ asPlainText: true }}
-              options={types.map(type => ({ label: type, value: type }))}
-            />
-          );
-        }}
-      </UseField>
-    </FormRow>
+    <UseField
+      config={typeConfig}
+      defaultValue={initialType}
```

Comment:
Just a note: by setting the defaultValue manually you lose the benefit to read it from the form `defaultValue` object (used in "Edit" mode). Not sure if this was intentional so leaving a comment.